### ISSUE TITLE
Feature request: Sorting of starred stations by drag and drop

### ISSUE THREAD

**Issue by yaaghu** [tuner-labs/tuner#109]
I often discover yet another station of a specific genre, which is put automatically at the end of the list of starred stations.  Being able to sort that list by personal preferences through drag and drop would come in very handy.

**Comment by th0razin3** [comment]
Second this, this is a useful feature.

**Comment by sshermz** [comment]
Third this. Please stop the entropy.

Even if not exposed as configurable setting (which would be the ideal), at the very least, added stations should be sorted alphabetically.

**Comment by th0razin3** [comment]
Yeah, a sorting option would be ideal, but for the time being, just drag and drop will do.

**Comment by sshermz** [comment]
The problem is that it becomes very difficult to find specific stations if you have many favorites (like a couple dozen or more).

I'm opening a new issue to add sorted favs by default, which is a trivial code change (I already did it in my local copy).

Sort options or d&d would be nice, but these would take more effort. And this ticket (109) is on the latter, so sorting by default is actually a different issue.

**Comment by th0razin3** [comment]
Why not submit a PR.

**Comment by sshermz** [comment]
This discussion belongs in issue 222, but will answer here since you asked here...

I wasn't able to get flatpak working on my system (Linux Mint 21.3), but perhaps I'll try again sometime.

Are PR's being accepted? There are currently two old, open PR's.
